invoke.py

```diff
@@ -0,0 +1,68 @@
+# This will be used to prefix all docker objects (network, images, containers)
+project_name = 'app'
+# This is the root domain where the app will be available
+# The "frontend" container will receive all the traffic
+root_domain = project_name + '.test'
```

Comment:
Tiens question comme ça, pourquoi .test alors que .localhost renvoit vers la machine locale sans toucher au `/etc/hosts` ?

Reply:
Good question, but out of scope of this PR. Feel free to open an issue or a PR to discuss about that

Reply:
For the record, we must not use .localhost TLD. This will fail for people are using a VM (ie. Mac and Windows users), so their docker is not available at localhost